public/index.html

```diff
@@ -3,11 +3,47 @@
   <head>
     <meta charset="utf-8">
     <title>Hoodie Account</title>
+    <link rel="stylesheet" href="/default/css/default.css">
     <script src="/hoodie/client.js"></script>
   </head>
   <body>
-    <p>
-       Open browser console and play with <code>hoodie.account</code>.
-    </p>
+
+    <div data-hide="false" class="generic-loader-wrap"></div>
+
+    <div data-hide="true" class="input-forms">
```

Comment:
@Charlotteis semantically speaking, I think I get your point. I'll make the relevant changes soon.